airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/models.py

```diff
@@ -1,11 +1,14 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 
 import os
-from typing import Dict, Set
+from typing import Dict, List, Set
 
 from pydantic import BaseModel, Field, validator
 
 
+DEFAULT_PYTHON_VERSION = "3.10.12"
```

Comment:
I'd rather not set a default version and require all pyproject.toml file for our internal packages to declare a python version. The less `airbyte-ci` knows about package specificities the happier I am.

Reply:
Like it too, will do.